core/arch/aarch64/codec.h

```diff
@@ -30,8 +30,8 @@
  * DAMAGE.
  */
 
-#ifndef CODEC_H
-#define CODEC_H 1
+#ifndef CODEC_H_
```

Comment:
Agreed. I just used `_H_` to detect include guards, but if we look for the special comment to detect whole-file ifdefs we don't really have to change the include guard names.